client/src/helpers/constants.js

```diff
@@ -33,12 +27,13 @@ export const convertResponseToContentObject = (data, userId) => {
     director: data.director,
     thumbnailUrl: data.thumbnail[0].thumbnailUrl,
     trailerUrl: data.trailer[0].trailerUrl,
-    contentUrl: data.content.contentURL,
+    contentUrl: data.contentMovie.movieUrl,
+    countryOrigin: data.originCountry,
     genres: data.genres,
-    rating: data.rating,
-    isLiked: isLiked,
-    isDisliked: isDisliked,
-    contentDuration: getFormatedDuration(data.content.contentDuration),
+    rating: data.maturityRating,
+    isLiked: data.likes.find((item) => item === userId) ? true : false,
```

Comment:
This will be handled, wait for the update

Reply:
Can we take it later, when mang is done with update?